### PR TITLE
Add r-rsconnect r-rsqlite r-rversions r-sandwich r-scales r-seacarb r…

### PR DESCRIPTION
**Pull request by lpantano**
r-rsconnect
r-rsqlite
r-rversions
r-sandwich
r-scales
r-seacarb
r-selectr

xref: https://github.com/conda-forge/staged-recipes/issues/2009
ping @johanneskoester

### CODE REVIEW

**Comment by lpantano** [comment]
@bgruening can you help me with this, please?

`Selectors are suggested to take a <two spaces>#<one space>[<expression>] form.` 

Just what to change, I will do the changes.

Thanks

File: recipes/r-rsqlite/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+requirements:
+  build:
+    - r-base
+    - r-bh
+    - r-dbi >=0.4_9
+    - r-rcpp
+    - r-memoise
+    - r-plogr
+    - posix               # [win]
+    - {{native}}toolchain # [win]
```
Comment (bgruening) [inline recipes/r-rsqlite/meta.yaml]: I think you need here two spaces before the `#`.
How do you generate these recipes? With a skeleton? If so please update your conda-build.
I have also a script to assist you if you like.

**Comment by bgruening** [comment]
I think you need here two spaces before the #.
How do you generate these recipes? With a skeleton? If so please update your conda-build.
I have also a script to assist you if you like.

**Comment by lpantano** [comment]
Thanks @bgruening,

The scripts you are using are the ones posted on 2009? (`build_helpers.tar.gz`)

I am using that, I think the problem was the conda-build, I thought I updated, but not.

**Comment by bgruening** [comment]
Yes, latest version is here: https://github.com/bgruening/conda_r_skeleton_helper

**Comment by lpantano** [comment]
thanks @bgruening !

**Comment by bgruening** [comment]
Sure thing! Thanks for helping!

**Comment by bgruening** [comment]
You can skip the win r-rsconnect build by adding `skip: true  # [win]`.
please also include a comment that r-rcurl is not available under windows.

File: recipes/r-rsconnect/build.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+mv DESCRIPTION DESCRIPTION.old
+grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
```
Comment (johanneskoester) [inline recipes/r-rsconnect/build.sh]: Above two lines should be omitted. See the cleanup scripts of bgruening.

File: recipes/r-rsqlite/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  build:
+    - r-base
+    - r-bh
+    - r-dbi >=0.4_9
+    - r-rcpp
+    - r-memoise
+    - r-plogr
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
```
Comment (johanneskoester) [inline recipes/r-rsqlite/meta.yaml]: If gcc is a build dependency, `libgcc  # [not win]` should be a run dependency.

Comment (lpantano) [inline recipes/r-rsqlite/meta.yaml]: fixing. thanks!

File: recipes/r-scales/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    - r-rcolorbrewer
+    - r-rcpp
+    - r-dichromat
+    - r-labeling
+    - r-munsell >=0.2
+    - r-plyr
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+    - libgcc               # [not win]
```
Comment (bgruening) [inline recipes/r-scales/meta.yaml]: libgcc should be a runtime dependency, not build dependency

**Comment by isuruf** [comment]
Let's merge this once staged-recipes is clear.

**Pull request merged**